Map/RoomBase.gd

```diff
@@ -4,15 +4,15 @@ class_name RoomBase
 ##
 ## Holds basic functionality for a room on the map
 
-@export var light_level: Enums.LightLevel:
+@export var light_data: LightData:
 	set(value):
-		light_level = value
-@export var has_torch: bool = false
+		light_data = value
```

Comment:
it's a bit strange to have such a setter when we don't have a setter in LightData

Reply:
Good catch, forgot to clean it up once I changed it to light_data